### ISSUE TITLE
Could not find a package configuration file provided by "geodesy"

### ISSUE THREAD

**Issue by ArghyaChatterjee** [CesMak/rviz_pics#1]
Please help!!!
What should I do now?
when I tried to install this package according to your instruction the following error occurs:

```CMake Error at /opt/ros/kinetic/share/catkin/cmake/catkinConfig.cmake:83 (find_package):
  Could not find a package configuration file provided by "geodesy" with any
  of the following names:

    geodesyConfig.cmake
    geodesy-config.cmake

  Add the installation prefix of "geodesy" to CMAKE_PREFIX_PATH or set
  "geodesy_DIR" to a directory containing one of the above files.  If
  "geodesy" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  rviz_pics/CMakeLists.txt:43 (find_package)


-- Configuring incomplete, errors occurred!
See also "/home/arghya/catkin_ws/build/CMakeFiles/CMakeOutput.log".
See also "/home/arghya/catkin_ws/build/CMakeFiles/CMakeError.log".
Invoking "cmake" failed

**Comment by CesMak** [comment]
Hey try to install geodesy package:

`sudo apt-get install ros-kinetic-geodesy`

**Comment by ArghyaChatterjee** [comment]
Thanks @CesMak, I managed to get rid of the error. However, when I tried to run the following command:
``` roslaunch aruco_detector_ocv detector.launch ``` from your tutorial at https://github.com/CesMak/aruco_detector_ocv, I get the following error:
```arghya@arghya-Aspire-VN7-571G:~/catkin_ws/src$ roslaunch tf_mapping start_tf_mapping.launch
... logging to /home/arghya/.ros/log/b0c60b80-a887-11e9-a050-5c93a2ade090/roslaunch-arghya-Aspire-VN7-571G-5703.log
Checking log directory for disk usage. This may take awhile.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

started roslaunch server http://arghya-Aspire-VN7-571G:41149/

SUMMARY
========

PARAMETERS
 * /aruco_detector_ocv/blur_window_size: 7.0
 * /aruco_detector_ocv/camera: /usb_cam/image_raw
 * /aruco_detector_ocv/camera_info: /camera/camera_info
 * /aruco_detector_ocv/dictionary_name: DICT_4X4_1000
 * /aruco_detector_ocv/image_height: 1080.0
 * /aruco_detector_ocv/image_width: 1920.0
 * /aruco_detector_ocv/marker_size: 0.05
 * /aruco_detector_ocv/min_prec_value: 50.0
 * /aruco_detector_ocv/num_detected: 25.0
 * /aruco_detector_ocv/show_detections: True
 * /aruco_detector_ocv/tf_prefix: marker_id
 * /rosdistro: kinetic
 * /rosversion: 1.12.14
 * /usb_cam/camera_frame_id: usb_cam
 * /usb_cam/image_height: 1080
 * /usb_cam/image_width: 1920
 * /usb_cam/io_method: mmap
 * /usb_cam/pixel_format: yuyv
 * /usb_cam/video_device: /dev/video1

NODES
  /
    aruco_detector_ocv (aruco_detector_ocv/node)
    camera_info (rostopic/rostopic)
    rviz (rviz/rviz)
    tf_mapping_node (tf_mapping/tf_mapping_node)
    usb_cam (usb_cam/usb_cam_node)

auto-starting new master
process[master]: started with pid [5714]
ROS_MASTER_URI=http://localhost:11311

setting /run_id to b0c60b80-a887-11e9-a050-5c93a2ade090
process[rosout-1]: started with pid [5727]
started core service [/rosout]
process[usb_cam-2]: started with pid [5730]
process[camera_info-3]: started with pid [5743]
process[aruco_detector_ocv-4]: started with pid [5746]
process[tf_mapping_node-5]: started with pid [5747]
process[rviz-6]: started with pid [5748]
[ INFO] [1563363636.331727002]: using default calibration URL
[ INFO] [1563363636.331829640]: camera calibration URL: file:///home/arghya/.ros/camera_info/head_camera.yaml
[ INFO] [1563363636.333880607]: Starting 'head_camera' (/dev/video1) at 1920x1080 via mmap (yuyv) at 30 FPS
[ WARN] [1563363636.493235463]: unknown control 'focus_auto'

[ INFO] [1563363636.946851920]: camera model is computed
[usb_cam-2] process has died [pid 5730, exit code -11, cmd /home/arghya/catkin_ws/devel/lib/usb_cam/usb_cam_node __name:=usb_cam __log:=/home/arghya/.ros/log/b0c60b80-a887-11e9-a050-5c93a2ade090/usb_cam-2.log].
log file: /home/arghya/.ros/log/b0c60b80-a887-11e9-a050-5c93a2ade090/usb_cam-2*.log

**Comment by CesMak** [comment]
Did you check the video:

`https://www.youtube.com/watch?v=1jss-v16y4w`

Do you try to use your camera or try to load stored images?

Check also if you get your camera to output images without my code so just try to get images from your camera in.

**Comment by ArghyaChatterjee** [comment]
After watching your video, I tried to start the tf_mapping node. The error shows up something like this. Still the no image issue persists.
```roslaunch tf_mapping start_tf_mapping.launch
... logging to /home/arghya/.ros/log/315a8b4e-a910-11e9-a050-5c93a2ade090/roslaunch-arghya-Aspire-VN7-571G-22339.log
Checking log directory for disk usage. This may take awhile.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

started roslaunch server http://arghya-Aspire-VN7-571G:35869/

SUMMARY
========

PARAMETERS
 * /aruco_detector_ocv/blur_window_size: 7.0
 * /aruco_detector_ocv/camera: /usb_cam/image_raw
 * /aruco_detector_ocv/camera_info: /camera/camera_info
 * /aruco_detector_ocv/dictionary_name: DICT_4X4_1000
 * /aruco_detector_ocv/image_height: 1080.0
 * /aruco_detector_ocv/image_width: 1920.0
 * /aruco_detector_ocv/marker_size: 0.05
 * /aruco_detector_ocv/min_prec_value: 50.0
 * /aruco_detector_ocv/num_detected: 25.0
 * /aruco_detector_ocv/show_detections: True
 * /aruco_detector_ocv/tf_prefix: marker_id
 * /rosdistro: kinetic
 * /rosversion: 1.12.14
 * /usb_cam/camera_frame_id: usb_cam
 * /usb_cam/image_height: 1080
 * /usb_cam/image_width: 1920
 * /usb_cam/io_method: mmap
 * /usb_cam/pixel_format: yuyv
 * /usb_cam/video_device: /dev/video1

NODES
  /
    aruco_detector_ocv (aruco_detector_ocv/node)
    camera_info (rostopic/rostopic)
    rviz (rviz/rviz)
    tf_mapping_node (tf_mapping/tf_mapping_node)
    usb_cam (usb_cam/usb_cam_node)

ROS_MASTER_URI=http://localhost:11311

process[usb_cam-1]: started with pid [22356]
process[camera_info-2]: started with pid [22357]
process[aruco_detector_ocv-3]: started with pid [22358]
process[tf_mapping_node-4]: started with pid [22366]
process[rviz-5]: started with pid [22380]
[ WARN] [1563422654.794036017]: Shutdown request received.
[ WARN] [1563422654.794101879]: Reason given for shutdown: [new node registered with same name]
[ INFO] [1563422655.092574726]: using default calibration URL
[ INFO] [1563422655.092809982]: camera calibration URL: file:///home/arghya/.ros/camera_info/head_camera.yaml
[ INFO] [1563422655.094112724]: Starting 'head_camera' (/dev/video1) at 1920x1080 via mmap (yuyv) at 30 FPS
[ INFO] [1563422655.112996642]: camera model is computed
terminate called after throwing an instance of 'class_loader::LibraryUnloadException'
  what():  Attempt to unload library that class_loader is unaware of.
[ WARN] [1563422655.255067630]: unknown control 'focus_auto'

0x1e28190 void QWindowPrivate::setTopLevelScreen(QScreen*, bool) ( QScreen(0x12c5b60) ): Attempt to set a screen on a child window.
0x1e27c90 void QWindowPrivate::setTopLevelScreen(QScreen*, bool) ( QScreen(0x12c5b60) ): Attempt to set a screen on a child window.
0x1e2fb90 void QWindowPrivate::setTopLevelScreen(QScreen*, bool) ( QScreen(0x12c5b60) ): Attempt to set a screen on a child window.
0x1713c50 void QWindowPrivate::setTopLevelScreen(QScreen*, bool) ( QScreen(0x12c5b60) ): Attempt to set a screen on a child window.
0x232ed40 void QWindowPrivate::setTopLevelScreen(QScreen*, bool) ( QScreen(0x12c5b60) ): Attempt to set a screen on a child window.
0x208fa30 void QWindowPrivate::setTopLevelScreen(QScreen*, bool) ( QScreen(0x12c5b60) ): Attempt to set a screen on a child window.
0x232f430 void QWindowPrivate::setTopLevelScreen(QScreen*, bool) ( QScreen(0x12c5b60) ): Attempt to set a screen on a child window.
0x2349b80 void QWindowPrivate::setTopLevelScreen(QScreen*, bool) ( QScreen(0x12c5b60) ): Attempt to set a screen on a child window.
0x2392f50 void QWindowPrivate::setTopLevelScreen(QScreen*, bool) ( QScreen(0x12c5b60) ): Attempt to set a screen on a child window.
[usb_cam-1] process has died [pid 22356, exit code -11, cmd /home/arghya/catkin_ws/devel/lib/usb_cam/usb_cam_node __name:=usb_cam __log:=/home/arghya/.ros/log/315a8b4e-a910-11e9-a050-5c93a2ade090/usb_cam-1.log].
log file: /home/arghya/.ros/log/315a8b4e-a910-11e9-a050-5

**Comment by ArghyaChatterjee** [comment]
I have calibrated my camera using this tutorial : http://ros-developer.com/2017/04/23/camera-calibration-with-ros/
The main problem I think is starting the usb_cam node. After I follwed your tutorial, I am unable to get the image. I tried to open my camera using usb_cam-test.launch file in usb_cam package. It works fine.
![Screenshot from 2019-07-18 10-22-30](https://user-images.githubusercontent.com/28845357/61428932-0cdb1500-a946-11e9-9d90-06831368286e.png)
